auth/openid_connect.md

```diff
@@ -65,6 +63,7 @@ following:
 | Client Protocol                    | openid-connect                                    |
 | Access Type                        | confidential                                      |
 | Valid Redirect URIs                | <https://<miq-appliance>>                         |
+|                                    | <https://<miq-appliance>/ui/service/>             |
```

Comment:
I suggest adding a note that the trailing / is required.

`Note: The trailing / for the `/ui/sevice/` entry is required.`